components/quick_input.jsx

```diff
@@ -141,7 +143,7 @@ export default class QuickInput extends React.PureComponent {
             {clearable && value && this.props.onClear &&
                 <div
                     className='input-clear visible'
-                    onClick={this.onClear}
+                    onMouseDown={this.onClear}
```

Comment:
0/5, but maybe `onMouseUp` would better match a click here?

Reply:
Not in this case because the blur action is tied to mouseDown, I tried both :P